dashboard/app/controllers/levels_controller.rb

```diff
@@ -234,7 +234,8 @@ def level_params
       :dsl_text,
       :encrypted,
       {concept_ids: []},
-      {soft_buttons: []}
+      {soft_buttons: []},
+      {examples: []}
```

Comment:
Do we need to address http://stackoverflow.com/questions/8929230 like we do for `soft_buttons` below?